### PR TITLE
CI: First changes to try to fix CI.

### PR DESCRIPTION
**Pull request by zssherman**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
       matrix:
         python-version: ["3.9", "3.10", "3.11", "3.12"]
         os: [macOS, ubuntu, Windows]
-        inlcude:
+        include:
           - os: macos-latest
             PLAT: arm64
```
Comment (mgrover1) [inline .github/workflows/ci.yml:39]:
```suggestion
            PLAT: arm64
            micromamba-version: 1.5.10
```

Comment (mgrover1) [reply .github/workflows/ci.yml:39]: @zssherman - try specifying the last version before the 2.0 release

**Comment by zssherman** [comment]
Hmm confused on why the datarose plot is not working now....

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -34,9 +34,10 @@ jobs:
...
+            micromamba-version: 1.5.10
             INTERFACE64: ""
```
Comment (mgrover1) [inline .github/workflows/ci.yml:41]:
```suggestion
            INTERFACE64: ""
            post-cleanup: "all"
```

**Review by mgrover1 — APPROVED** [review]

**Pull request merged**